browser-extension/plugin/package.json

```diff
@@ -5,7 +5,7 @@
     "scripts": {
         "prepare": "cd ../.. && husky install \"browser-extension/plugin/.husky\"",
         "test": "echo \"Error: no test specified\" && exit 1",
-        "test-jest": "jest",
+        "api-test": "jest",
```

Comment:
lets use consistent convetion. we've been using ":" as the delimiter in the other script names. lets call it test:api instead of api-test

Reply:
will do